op-supervisor/supervisor/backend/backend.go

```diff
@@ -476,6 +482,12 @@ func (su *SupervisorBackend) CheckMessages(
 	return nil
 }
 
+func (su *SupervisorBackend) CheckMessages(
+	messages []types.Message,
+	minSafety types.SafetyLevel) error {
+	return su.CheckMessagesV2(messages, minSafety, types.ExecutingDescriptor{Timestamp: math.MaxUint64})
```

Comment:
With `math.MaxUint64`, isn't the initiating message always much older, and thus expired?

This would brick the current devnet interop usage as soon as the supervisor is updated, until op-geth is updated.
I think we should maintain the old behavior in the V1 API.